packages/components/src/components/u-i-loader/ui-loader.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import classNames from 'classnames';
+import Loadable from 'react-loadable';
 
 type TUILoader = {
     className?: string;
     classNameBlock?: string;
-};
+} & Loadable.LoadingComponentProps;
```

Comment:
Hi @farzin-deriv, This was included here, because `Loadable` from `react-loadable` expects the value of the `loading` property to be of `type` `ComponentType<LoadingComponentProps>` and the current `type` definition for the`UILoader` doesn't meet those requirements

Reply:
@jim-deriv Thanks for the explanation 🌹🙇🏻